### PR TITLE
Add "get total flows" endpoint

### PR DESCRIPTION
**Pull request by konstantinos-papadopoulos**
For testing new functionality
1. launch ie three workers with IPs 192.168.100.21,192.168.100.22,192.168.100.23
2. bin/deploy --json-config config/config.json
3. bin/handlers/init_topos --json-config config/config.json
4. bin/handlers/start_topos --json-config config/config.json

add some flows to the switches of each worker ie if 10 switches per worker and number of workers =3

**Worker 1**
```bash
sudo ovs-ofctl -O OpenFlow13 add-flow 1 'priority=2,icmp,actions=CONTROLLER:65535'
sudo ovs-ofctl -O OpenFlow13 add-flow 1 'priority=2,ip,actions=CONTROLLER:65535'
sudo ovs-ofctl -O OpenFlow13 add-flow 1 'priority=2,arp,actions=CONTROLLER:65535'
```

**Worker 3**
```bash
sudo ovs-ofctl -O OpenFlow13 add-flow 21 'priority=2,icmp,actions=CONTROLLER:65535'
sudo ovs-ofctl -O OpenFlow13 add-flow 22 'priority=2,ip,actions=CONTROLLER:65535'
sudo ovs-ofctl -O OpenFlow13 add-flow 23 'priority=2,arp,actions=CONTROLLER:65535'
```

5. bin/handlers/get_flows --json-config config/config.json

Results should be

```bash
["{\"dpid-0\": 3}", "{\"dpid-1\": 0}", "{\"dpid-2\": 3}"]
```

### CODE REVIEW

File: bin/handlers/get_flows [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""Get the number of started switches
```
Comment (anastop) [inline bin/handlers/get_flows]: is this valid?

File: config/config.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
 {
```
Comment (anastop) [inline config/config.json]: should this file change? it is used in the documentation as is.

Read the whole documentation to see possible things that could break with the pull request.

File: multi/worker.py [hunk 3]
```diff
@@ -95,6 +95,18 @@ def get_switches():
...
+@bottle.route('/get_flows', method='POST')
+def get_flows():
+    """
+    Calls the get_flows() method of the current topology object to get the
+    current number of flows installed on the switches.
+
+    Returns
+        str: A JSON string with dpid_offset/number_of_switches key/value pairs
+    """
+    dpid_key = 'dpid-{0}'.format(MININET_TOPO._dpid_offset)
```
Comment (anastop) [inline multi/worker.py]: this shouldn't be `dpid`. `dpid` is a unique ID for the switches. We need a worker id here.

**Comment by konstantinos-papadopoulos** [comment]
Under file ''multinet/multinet_src/multi/worker.py' l.107
```python
> dpid_key = 'number-of-flows-on-worker-{0}'.format(MININET_TOPO._dpid_offset)
```
Results should be

```bash
["{\"number-of-flows-on-worker-0\": 3}", "{\"number-of-flows-on-worker-1\": 0}", "{\"number-of-flows-on-worker-2\": 3}"]
```
Should now be ok

**Pull request merged**